### PR TITLE
Rspec Test

### PR DESCRIPTION
**Pull request by Juakata**

### CODE REVIEW

File: spec/methods_spec.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  describe '#my_each' do
+    it 'returns all elements in the array' do
+      expect(test_array_1.my_each { |e| e }).to eql(test_array_1)
+    end
+  end
+
+  describe '#my_each_with_index' do
+    it 'returns all elements with the index in the array' do
+      expect(test_array_1.my_each_with_index { |_e, i| i }).to eql(test_array_1)
```
Comment (bolah2009) [inline spec/methods_spec.rb]: #### `REQUIRED`

Consider deleting unused argument.

```suggestion
      expect(test_array_1.my_each_with_index { |i| i }).to eql(test_array_1)
```

**Review by bolah2009 — CHANGES_REQUESTED** [review]
# Awesome Job Team :clap: 

Some tests are not covered. I commented out parts of the code and the test is still passing. Add tests for `my_none?` and `multiply_els`. Also, check other cases listed in [this issue](https://github.com/dev1980/Enumerable_Methods/issues/4)

Check out the review comments and issue (#4). :blush: 

Fix this and submit another code review request. :ok_hand: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @Juakata , @dev1980 
#### Great work guys ! :+1: 
Just one thing left to do, kindly attend to my comment.

_Kindly re-submit for review after making all this change :slightly_smiling_face: Thumps -up await you_

**Review by bellom — APPROVED** [review]
Hi @Juakata @dev1980,
**Great work making all the necessary corrections from the previous review.**

**NOTED:**  Always read the pathwright instructions before starting the project to avoid the need of sending another code review, according to the pathwright you are to use `Git Flow` not `GitHub Flow`.
Short Explanation Hint: 
`GitHub Flow`: Pull Requests is raise from `feature branches to the master`.
`Git Flow`: Pull Requests is raise from `feature branches to the development`.
**Kindly make use of `Git Flow` in the next project !**
## Congratulation :+1:  
_Go ahead and merge to master and don't forget to submit project completion form.
Goodluck in your next project :slightly_smiling_face:_

**Pull request merged**